apps/studio/src/components/CmsSidebar/CmsSideNav.tsx

```diff
@@ -0,0 +1,192 @@
+import { Suspense } from "react"
+import { useRouter } from "next/router"
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Flex,
+  Skeleton,
+  Text,
+  useDisclosure,
+  VStack,
+} from "@chakra-ui/react"
+import { Menu } from "@opengovsg/design-system-react"
+import { ResourceType } from "@prisma/client"
```

Comment:
chromatic is failing the build because you are importing a type that has not been generated yet. Suggest importing from the source of truth 

```suggestion
import { ResourceType } from "~prisma/generated/generatedEnums"
```

Reply:
xxlb